### ISSUE TITLE
Undefined property: stdClass::$newest ... on line 206

### ISSUE THREAD

**Issue by EnterpriseBranding** [Freemius/wordpress-sdk#173]
**Actual Behavior**:
 -  `What is the issue? (*)`
I get the following with debug on. `Notice: Undefined property: stdClass::$newest in start.php on line 206.` Then naturally I get `Notice: Trying to get property of non-object in start.php on line 206
` I briefly file searched before submitting this and didn't find anywhere else this is used. Is it just a left over or is it needed? I know it is to be part of the stdClass() but I'm not sure where to view the code of that class, as it would be my first time reading on the built-in class like that in PHP.

I development I run a consistent debug @ `wp-admin/admin.php?debug=on` and this is where this notice shows. Could it be my PHP version?

 -  `What is the expected behavior?`
Notice error shouldn't be.

**Versions**: (*)
 -  `Freemius SDK Version:`
Latest
 -  `WordPress Version:`
Latest
 -  `PHP Version:`
PHP Version 5.5.9-1ubuntu4.21

**Plugin / Theme**: (*)
- Plugin

**Comment by vovafeldman** [comment]
@FearVsFear sorry for the late response. Is it possible that the environment that has thrown the error is a replica of another environment? Also, what's the SDK version? (latest is kind of relative)

**Comment by FearTheRevolution** [comment]
@vovafeldman 

Thanks for still responding. No problem. I have kept it deactivated in development since actually and I'm still developing. So no rush actually.

Hmmm... actually I believe it might be a replica, but I'm not sure. It's been awhile since I created the environment. Though, even if it is I don't believe Freemius was activated on the previous. Is there some kind of cached data I can look into?

This is SDK Version `1.2.2.9`.
